### PR TITLE
Enhancing data quality in data science and machine learning

### LINKED ISSUE #4558: Enhancing data quality in data science and machine learning
#### Proposed title of article
Enhancing data quality in data science and machine learning 

#### Proposed article introduction
Every data scientist and Machine Learning evangelist's ultimate goal is to build a better model with higher predictive accuracy. However, data may be the issue when it comes to fine-tuning hyperparameters or refining modeling methods. 
No matter how powerful a Machine Learning model is, if the data is typically of poor quality, the results will always be substandard at best. This article seeks to provide ways to enhance data quality for better results.

#### Key takeaways 

the reader should be able to
- understand data and it's role in machine learning and artificial intelligence
- understand the importance of working with quality data
- know some techniques to achieve quality data 

#### Article quality
This article is well elaborated with less technical terms to enable beginners and experts understand ways to enhance data quality in data science and machine learning.

### PR DESCRIPTION
**Pull request by Doggedben000**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #4558

### CODE REVIEW

**Comment by Kayere** [comment]
Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept. Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content. Be sure to see our resources page to see more info on plagiarism and what is considered as such.

I will be closing this PR to allow you some time to make the changes. Please feel free to open a new PR once the changes are done.

[Enhancing data quality in data science and machine learning #4668 Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/7453998/Enhancing.data.quality.in.data.science.and.machine.learning.4668.Copyleaks.Report.pdf)

**Comment by srishilesh** [comment]
@Doggedben000 

Please ping me in the new PR, with all the changes done.

Thanks.

**Comment by srishilesh** [comment]
@Doggedben000 

Do we have any updates?

Just wanted to check if you need any help.
